diff -ur aumix-2.8.orig/src/xaumix aumix-2.8/src/xaumix
--- aumix-2.8.orig/src/xaumix	2002-03-28 10:37:27.000000000 +0100
+++ aumix-2.8/src/xaumix	2004-07-12 19:45:19.488147456 +0200
@@ -43,6 +43,9 @@
 # 	GnomeUI-WARNING **: Could not open help topics file NULL
 		if test -x $dir/gnome-terminal; then
 			GEOMETRY='--geometry 79x${LINES}'
+# Gnome-terminal tries to parse '-C' as it's own option. And fails.
+# Fortunatelly aumix can work without '-C ansi'.
+			SCHEME=""
 			NAME="--name=aumix"
 			TITLE="-t aumix"
 			XTERM="$dir/gnome-terminal --name aumix"
