diff -ur aumix-2.8.orig/configure.in aumix-2.8/configure.in
--- aumix-2.8.orig/configure.in	2002-11-25 02:04:16.000000000 +0100
+++ aumix-2.8/configure.in	2004-07-12 19:29:45.135190696 +0200
@@ -61,30 +61,30 @@
 	[Define this if your ncurses library contains the use_default_colors
 	routine and your terminal can use transparent colors.])])
     AC_CACHE_CHECK([for private member _use_keypad in WINDOW],
-      aumix_cv_struct_window_usekeypad,
+      [aumix_cv_struct_window_usekeypad],
       [AC_TRY_COMPILE([#ifdef HAVE_NCURSES_H
 #include <ncurses.h>
 #else
 #include <curses.h>
-#endif], [WINDOW w; w._use_keypad;],
-      aumix_cv_struct_window_usekeypad=yes, aumix_cv_struct_window_usekeypad=no)])
+#endif
+], [WINDOW w; w._use_keypad;],
+      [aumix_cv_struct_window_usekeypad=yes],[ aumix_cv_struct_window_usekeypad=no])])
     if test $aumix_cv_struct_window_usekeypad = yes; then
       AC_DEFINE(HAVE_USEKEYPAD, 1,
 	[Define this if your ncurses library has the member _use_keypad
 	in structure WINDOW.])
     fi
   fi
-  AC_ARG_WITH(gpm,
-    [  --without-gpm           Compile only without GPM support],
-    AC_MSG_RESULT([Compiling without GPM support]),
+])
+  AC_ARG_WITH(gpm,[  --without-gpm           Compile only without GPM support],[
+    AC_MSG_RESULT([Compiling without GPM support])],[
     dnl Checks for GPM library.
     AC_CHECK_LIB(gpm, Gpm_Repeat)
-  )
-  AC_ARG_WITH(sysmouse,
-    [  --without-sysmouse      Compile only without sysmouse support],
-    AC_MSG_RESULT([Compiling without sysmouse support]),
+  ])
+  AC_ARG_WITH(sysmouse,[  --without-sysmouse      Compile only without sysmouse support],[
+    AC_MSG_RESULT([Compiling without sysmouse support])],
     dnl Checks for sysmouse availability.
-    AC_CHECK_HEADERS(sys/consio.h machine/console.h, break)
+    dnl AC_CHECK_HEADERS([sys/consio.h machine/console.h], [break])
     AC_CACHE_CHECK([for CONS_MOUSECTL],
       aumix_cv_cons_mousectl,
       [AC_EGREP_CPP([A mouse is a mouse is a mouse],
@@ -96,14 +96,15 @@
 #endif
 #ifdef CONS_MOUSECTL
 A mouse is a mouse is a mouse
-#endif], aumix_cv_cons_mousectl=yes, aumix_cv_cons_mousectl=no)])
+#endif
+],[ aumix_cv_cons_mousectl=yes],[ aumix_cv_cons_mousectl=no])])
     if test $aumix_cv_cons_mousectl = yes; then
       AC_DEFINE(HAVE_SYSMOUSE, 1,
 	[Define this if you have sysmouse console mouse support and
 	want to use it.])
     fi
   )
-])
+
 AM_CONDITIONAL(CURSES, test "x$CURSLIB" != "x")
 AC_SUBST(CURSLIB)
 
